### PR TITLE
Issue 7 - Add survey_id to returned objects 

### PR DESCRIPTION
**Pull request by byerc**

### CODE REVIEW

File: test/test-config.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+'use strict';
```
Comment (trong-le) [inline test/test-config.js]: Shouldn't need this. Should just need the `requre('dotenv').config();`. Also, shouldn't need the `const config = { ... } export ..`. Should just be able to do
`export default { surveyId: ..., apiToken: ..., etc. };`

File: lib/index.js [hunk 2]
```diff
@@ -11,193 +17,262 @@ export default class SurveyGizmoSource extends Activity {
...
 
     async getSurveyList({ apiVersion, apiToken }) {
         const uri = `${apiVersion}/survey.json?api_token=${apiToken}`;
-        const resp = await this.apiGetRequest(uri);
+        const resp = await this.apiGetRequest(this.setupGetRequestParameters(uri));
+        //Not calling addSurveyIdToObject here because it's a special case
+        //Just appending an additional field with the same id for now
+        if(resp.result_ok === true && !isEmpty(resp.data)) {
+            resp.data.forEach((survey) => {
+                survey.survey_id = survey.id;
```
Comment (trong-le) [inline lib/index.js]: For these returned responses, we should really manipulate the response but make a copy of it, with the spread operator, and returning that instead.

File: lib/index.js [hunk 3]
```diff
@@ -11,193 +17,262 @@ export default class SurveyGizmoSource extends Activity {
...
-        return resp;
+        const resp = await this.apiGetRequest(this.setupGetRequestParameters(uri));
+        return this.addSurveyIdToObject(resp, surveyId);
+    }
+
+    async getAll(config){
+        const resp = this.getSurveyList(config);
+        if(resp.result_ok === true) {
+            resp.data.forEach((survey) => {
+
```
Comment (trong-le) [inline lib/index.js]: not doing anything with the surveys and not returning anything

File: lib/index.js [hunk 4]
```diff
@@ -11,193 +17,262 @@ export default class SurveyGizmoSource extends Activity {
...
+    }
+
+    setupGetRequestParameters(uri) {
+        return {
+            endpoint: uri,
+            uriOptions: ''
+        };
+    }
+
+    addSurveyIdToObject(resp, surveyId) {
```
Comment (trong-le) [inline lib/index.js]: same about not manipulating the response but returning a copy of it

File: lib/util.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+const flattenUrlOptions = (options) => {
+  const entries = Object.keys(options);
```
Comment (trong-le) [inline lib/util.js]: change all indentation to 4 spaces

File: lib/index.js (hunk 3 above)
Comment (byerc) [inline lib/index.js]: I shouldn't have included this. It's a separate issue, that's why I stopped working on it.

**Comment by byerc** [comment]
Fixed issues, please re-review.

**Comment by trong-le** [comment]
Also need to lock the Dockerfile aries-data import to the current aries-data version

**Pull request merged**